taxi/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=20, unique=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+    def __str__(self):
+        return self.username
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=100)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name="cars"
+    )
+    drivers = models.ManyToManyField(Driver, related_name="cars")
```

Comment:
Don't use user model explicitly. For this purpose use settings.AUTH_USER_MODEL. Also import settings from django.conf module instead of explicit settings file as you may have multiple settings files